portal/templates/redesign/teach_new/base_registering.html

```diff
@@ -4,11 +4,17 @@
 
 {% block secondaryButtons %}
 <a class="button--menu button--menu--secondary button--menu--teacher--active">School / Club</a>
+{% if onboarding_done %}
+<a class="button--menu button--menu--secondary button--menu--enabled">Rapid Router</a>
```

Comment:
I think your BEM syntax may have got a little muddled around here.  Is your menu an element of your button block?  Or is your button an element of your menu block?  Or are buttons and menus unrelated and unnecessarily tangled here?